Cython/CodeWriter.py

```diff
@@ -247,10 +248,37 @@ def visit_FuncDefNode(self, node):
         self.visit(node.body)
         self.dedent()
 
+    def visit_CFuncDefNode(self, node):
+        if 'inline' in node.modifiers:
+            return
```

Comment:
What? :)

Reply:
The first serveral lines are inherieted from `visit_CFuncDefNode` in `PxdWriter`, I'm not sure the aim indeed.